src/constants/FormElements.js

```diff
@@ -0,0 +1,22 @@
+export const createRegisterFormElements = () => {
```

Comment:
I wonder if this would be better placed in a test folder of sorts, but I think the way you guys structured the code base it seems fine, but it is something to note

Reply:
its just temporary, its cause this is used in 3 different views sub-folders, so I could move it into views/constants later